pkg/image/export/main.go

```diff
@@ -77,12 +77,14 @@ func run(systemChartPath string, imagesFromArgs []string) error {
 		data.K8sVersionInfo,
 	)
 
-	targetImages, err := img.GetImages(systemChartPath, imagesFromArgs, linuxInfo.RKESystemImages, img.Linux)
+	k3sUpgradeImages := getK3sUpgradeImages(data.K3S)
+
+	targetImages, err := img.GetImages(systemChartPath, k3sUpgradeImages, imagesFromArgs, linuxInfo.RKESystemImages, img.Linux)
 	if err != nil {
 		return err
 	}
 
-	targetWindowsImages, err := img.GetImages(systemChartPath, []string{getWindowsAgentImage()}, windowsInfo.RKESystemImages, img.Windows)
+	targetWindowsImages, err := img.GetImages(systemChartPath, k3sUpgradeImages, []string{getWindowsAgentImage()}, windowsInfo.RKESystemImages, img.Windows)
```

Comment:
k3sUpgradeImages should probably be empty. Even though a windows cluster will be a mixture of windows and linux nodes, the windows-images textfile is just to explain the images needed for the windows nodes I believe, as the linux ones will be downloaded anyway. See: https://rancher.com/docs/rancher/v2.x/en/installation/other-installation-methods/air-gap/populate-private-registry/#linux-steps
Could have been me but I thought I changed this.